### PR TITLE
Update to the most recent Geant4 patch releases

### PR DESCRIPTION
**Pull request by cpommranz**

### CODE REVIEW

**Comment by maxaehle** [comment]
Thanks for updating the Geant4 versions, it is certainly helpful to have those! I'd prefer not to change an existing set `vX.Y` of versions though because that might harm easy reproducibility. Instead, I've created new versions `vX.Y_latest` with your updates. If this looks good to you, I'll merge this PR. 

(GitHub allowed me to push to your master branch, probably because you allowed maintainers to edit this PR - thanks and feel free to revert these changes once the PR is merged.)

**Review by maxaehle — APPROVED** [review]

**Comment by cpommranz** [comment]
Looks good to me, thank you for editing! Keeping the existing versions untouched definitely makes sense.

**Pull request merged**
